sdk/trace/trace_test.go

```diff
@@ -192,10 +191,18 @@ func TestSampling(t *testing.T) {
 					sampled++
 				}
 			}
+			tolerance := 0.0
 			got := float64(sampled) / float64(total)
+
+			if tc.expect > 0 && tc.expect < 1 {
+				// See https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval
+				const z = 4.75342 // This should succeed 99.9999% of the time
```

Comment:
I'd prefer if you went with the Wilson score interval with continuity correction.

Joking, of course. ;)